migration/v0.18.x-v0.19.x/upgrade-apps.md

```diff
@@ -25,3 +49,9 @@
     ```bash
     bin/ck8s apply {sc|wc}
     ```
+
+1. Clean up after ODFE:
+
+    Run script `opensearch-migration-clean.sh`, this will delete deprecated parameters for ODFE in `sc-config.yaml` and `wc-config.yaml`.
+
+    Delete deprecated secrets for `elasticsearch` in `secrets.yaml`.
```

Comment:
Wouldn't it be nice to have this in the script as well?

Reply:
I'll see if I can add it.

Reply:
For inspiration, this is how we have done it previously
https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration/v0.7.x-v0.8.x/migrate-object-storage.sh#L42-L46

Reply:
Did you look into this anything yet?

Reply:
No, I haven't gone through the whole `migration-run` script yet, but when I do I'll add it.